### DISCUSSION TITLE
how to batch modify class ID quickly? Thanks.

### DISCUSSION THREAD

**Discussion by apple2345678** [systemsomicslab/MsdialWorkbench#122] [Q&A] [answered]
Excuse me. could I ask how to fill down the same class ID?  if samples are more than 100, I have to copy and paste many times. is there some simple method to batch modify class ID? Thank for your reply.
![modify class ID](https://github.com/systemsomicslab/MsdialWorkbench/assets/137306240/0a3efd7c-10ec-40d0-9f59-1aadef8436a3)

**Comment by kozo2** [comment, marked as answer]
Hi @apple2345678

So far we have not implemented the Excel autofill like functionality within MS-DIAL5.
Therefore, the current solution is
1. create the cells (what you want) with Excel/Google Sheets autofill featrue.
2. and copy the cells in Excel/Google Sheets.
3. and then paste the cells into MS-DIAL5.

**Reply by apple2345678** [reply]
Thank you for your reply.
